Building with plugins

Upon updating to the newest version of `iptb`, I tried to rebuild/reinstall. When I tried to run `iptb auto --type localipfs --count 3`, I got an error saying `Could not find plugin localipfs`. Upon running `make plugins`, I got the following output/error:

```
make -C plugins all                                                        
make[1]: Entering directory '/home/grish/.go/src/github.com/ipfs/iptb/plugins'          
make -C ipfs all                                                                       
make[2]: Entering directory '/home/grish/.go/src/github.com/ipfs/iptb/plugins/ipfs'   
gx-go rw                                                                          
ERROR: Error: build of rewrite mapping failed:                                              
package "go-multiaddr" not found. (dependency of ipfs-iptb-plugins)                              
make[2]: *** [Makefile:4: ipfslocal] Error 1                   
make[2]: Leaving directory '/home/grish/.go/src/github.com/ipfs/iptb/plugins/ipfs'    
make[1]: *** [Makefile:10: ipfs] Error 2                                        
make[1]: Leaving directory '/home/grish/.go/src/github.com/ipfs/iptb/plugins' 
make: *** [Makefile:9: plugins] Error 2
```

To fix this, I navigated to the `plugins` directory and ran `gx install`. There should probably be a check/command to handle this automatically.

Pinging @travisperson at the request of @frrist.

--- a/plugins/ipfs/Makefile
+++ b/plugins/ipfs/Makefile
@@ -1,12 +1,15 @@
 all: ipfslocal ipfsdocker
 
-ipfslocal:
+deps:
+	gx install
+
+ipfslocal: deps
 	gx-go rw
 	(cd local; go build -buildmode=plugin -o ../../localipfs.so)
 	gx-go uw
 CLEAN += localipfs.so
 
-ipfsdocker:
+ipfsdocker: deps
 	gx-go rw
 	(cd docker; go build -buildmode=plugin -o ../../dockeripfs.so)
 	gx-go uw

